### ISSUE TITLE
Navigation > Tags

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#552]
## Description

Tags and might probablly be used to enhance [badges](https://v5-dev--boosted.netlify.app/docs/5.0/components/badge/), or should it be a brand new component?

Note: Should be used in a11y-guidelines for component examples:  https://a11y-guidelines.orange.com/fr/web/exemples-de-composants/tags/ + https://a11y-guidelines.orange.com/fr/web/exemples-de-composants/tags/2/

## References

* **Orange Design System**: [Components > Navigation](https://system.design.orange.com/0c1af118d/p/44331c-navigation/b/9105e6)
  * Tags standard  — WEB-NAV-TAG-001
  * Tags with icon — WEB-NAV-TAG-002
  * Tags with remove icon — WEB-NAV-TAG-003
  * Tags with icon and remove icon — WEB-NAV-TAG-004

**Comment by Nurovek** [comment]
https://app.zeplin.io/project/5db823dcd517762c27b3d72f/screen/5db825889f92a52c57a20cdc

Is the part of the UI Kit where Tags can be found

**Comment by ffoodd** [comment]
Zeplin is private, I guess you should not link to it from GitHub :)
